luigi_pipeline/seqr_loading.py

```diff
@@ -18,14 +20,61 @@ class SeqrVCFToMTTask(HailMatrixTableTask):
     reference_mt_path = luigi.Parameter(description='Path to the matrix table storing the reference variants.')
     sample_type = luigi.ChoiceParameter(choices=['WGS', 'WES'], description='Sample type, WGS or WES', var_type=str)
     validate = luigi.BoolParameter(default=True, description='Perform validation on the dataset.')
+    dataset_type = luigi.ChoiceParameter(choices=['VARIANTS', 'SV'], default='VARIANTS', description='VARIANTS or SV.')
 
     def run(self):
         mt = self.import_vcf()
         if self.validate:
             self.validate_mt(mt, self.genome_version, self.sample_type)
-        # TODO: Modify MT.
+
+        mt = hl.split_multi(mt)
+        mt = HailMatrixTableTask.run_vep(mt, self.genome_version, self.vep_runner)
+        mt = self.derive_fields(mt, self.dataset_type)
+        mt.describe()
         mt.write(self.output().path)
 
+    @staticmethod
+    def derive_fields(mt, dataset_type):
+        """
+        Transforms a mt with fields derived from `vep`, `locus`, `alleles`, `position`, etc.
+        :param mt: matrix table with required fields (e.g. `vep`, `locus`, etc.)
+        :param dataset_type: either `VARIANTS` or `SV`
+        :return: mt with selected derived fields
+        """
+        mt = mt.annotate_rows(
+            sortedTranscriptConsequences=vep.get_expr_for_vep_sorted_transcript_consequences_array(mt.vep))
+        mt = mt.annotate_rows(
+            docId=variant_id.get_expr_for_variant_id(mt, 512),
```

Comment:
Just curious, how was 512 decided? Do we want that hardcoded?

Reply:
So this is copied over, and I assume they chose that based on some ES search string limit? 
My general heuristic is if it's used only once and pretty obvious, it can be a magic number.